plugins/database/quota/src/org/apache/cloudstack/api/response/QuotaStatementItemResponse.java

```diff
@@ -0,0 +1,143 @@
+//Licensed to the Apache Software Foundation (ASF) under one
+//or more contributor license agreements.  See the NOTICE file
+//distributed with this work for additional information
+//regarding copyright ownership.  The ASF licenses this file
+//to you under the Apache License, Version 2.0 (the
+//"License"); you may not use this file except in compliance
+//with the License.  You may obtain a copy of the License at
+//
+//http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing,
+//software distributed under the License is distributed on an
+//"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//KIND, either express or implied.  See the License for the
+//specific language governing permissions and limitations
+//under the License.
+package org.apache.cloudstack.api.response;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.apache.cloudstack.api.BaseResponse;
+
+import com.cloud.serializer.Param;
+
+public class QuotaStatementItemResponse extends BaseResponse {
+
+    @SerializedName("type")
+    @Param(description = "usage type")
+    private int usageType;
+
+    @SerializedName("accountid")
+    @Param(description = "account id")
+    private Long accountId;
+
+    @SerializedName("account")
+    @Param(description = "account name")
+    private String accountName;
+
+    @SerializedName("domain")
+    @Param(description = "domain id")
+    private Long domainId;
+
+    @SerializedName("name")
+    @Param(description = "usage type name")
+    private String usageName;
+
+    @SerializedName("unit")
+    @Param(description = "usage unit")
+    private String usageUnit;
+
+    @SerializedName("quota")
+    @Param(description = "quota consumed")
+    private BigDecimal quotaUsed;
+
+    @SerializedName("startdate")
+    @Param(description = "start date")
+    private Date startDate = null;
+
+    @SerializedName("enddate")
+    @Param(description = "end date")
+    private Date endDate = null;
+
+    public QuotaStatementItemResponse() {
```

Comment:
The default constructor declaration is unnecessary -- the compiler will generate it.